tests/test_conditional_flow_matcher.py

```diff
@@ -115,13 +123,14 @@ def test_fm(method, sigma, shape):
         x0, x1 = sample_plan(method, x0, x1, sigma)
 
     torch.manual_seed(TEST_SEED)
+    if test_eps:
+        # compute to get same t seed
+        eps = torch.randn_like(x0)
```

Comment:
where is this one used?

Reply:
okay I get it. Why don't you give eps instead of ret_eps in line 132 and drop the comment? esp and ret_eps are supposed the be the same.

Reply:
I think we want to test that they are the same? Make sure we didn't mess anything up.

Reply:
Ahh no we can't do that. It messes up the seeds for later inits.